default-sauce-browsers.json

```diff
@@ -9,27 +9,12 @@
     "platform":     "Windows 8.1",
     "version":      "11"
   },
-  {
-    "browserName":  "internet explorer",
-    "platform":     "Windows 7",
-    "version":      "10"
-  },
 
   {
     "browserName":  "safari",
     "platform":     "OS X 10.11",
     "version":      "9"
```

Comment:
Will add 10. Safari TP is [not listed](https://saucelabs.com/platforms) as supported on Saucelabs' site since they only support up to El Capitan. They don't seem to have a github repo in regards to their platforms. Contacted support to hopefully get some tracking on this issue soon.